tests/common-tests.h

```diff
@@ -6106,8 +6115,11 @@ TEST_CASE_METHOD(common_tests, "CLOB", "[core][clob]")
     sql << "select s from soci_test where id = 1", into(s2);
 
     CHECK(s2.value.size() == 0);
-
+#ifdef SOCI_ODBC_WIDE
+    s1.value = make_long_xml_string(3000);
+#else
```

Comment:
Could we handle this inside `make_long_xml_string()`, e.g. divide the length by 2 there in the wide char case?